README.rst

```diff
@@ -64,6 +68,25 @@ Install all the prerequisites using `dependencies.sh` script provided in the hom
    - Python 3.7 is used in this generator.
    - All the required tools need to be loaded into the environment before running this generator.
 
+Upgrading
+****************
+To upgrade packages used by the generators, the dependencies.sh script must be run, which can be found in the home folder of the cloned repository. 
+
+.. code-block:: bash
+
+    $ sudo ./dependencies.sh # Enter 'u' on being prompted for install or upgrade
```

Comment:
The comment is confusing here. What should be entered with when prompted?

Reply:
Ill rephrase that part